### PR TITLE
test(log): add toString unit test

### PR DESCRIPTION
**Pull request by 180909**
#### Which issue(s) this PR fixes (resolves / be part of):

part of https://github.com/go-kratos/kratos/issues/2354

### CODE REVIEW

File: contrib/log/aliyun/aliyun_test.go [hunk 1]
```diff
@@ -93,3 +94,30 @@ func TestLog(t *testing.T) {
 		t.Errorf("Log() returns error:%v", err)
 	}
 }
+
+func TestToString(t *testing.T) {
+	var tests = []struct {
```
[the remark this answers is not in the document]
Comment (180909) [reply contrib/log/aliyun/aliyun_test.go:99]: 不行，会报错的：too many values in struct literal

[the remark this answers is not in the document]
Comment (demoManito) [reply contrib/log/aliyun/aliyun_test.go:99]: image
没有问题

File: contrib/log/aliyun/aliyun_test.go [hunk 2]
```diff
@@ -93,3 +94,30 @@ func TestLog(t *testing.T) {
...
+		in  interface{}
+		out string
+	}{
+		{math.MaxFloat64, "179769313486231570000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000"},
```
Comment (demoManito) [inline contrib/log/aliyun/aliyun_test.go:103]: 这里需要换行，太长了，lint 有问题

File: contrib/log/aliyun/aliyun_test.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (180909) [reply contrib/log/aliyun/aliyun_test.go:99]: 抱歉，我看错了，我以为是把
`[]struct{
in interface{}
out string
}`
改成
`[]struct{}`

File: contrib/log/aliyun/aliyun.go [hunk 3]
```diff
@@ -126,7 +126,7 @@ func NewAliyunLog(options ...Option) Logger {
 	}
 }
 
-// toString 任意类型转string
+// toString any type to string
```
Comment (demoManito) [inline contrib/log/aliyun/aliyun.go:129]: convert any type to string

**Review by haiyux — APPROVED** [review]

**Comment by 180909** [comment]
ping？

**Review by Cluas — APPROVED** [review]

**Pull request merged**